Acrolinx quality improvements3: auditpol-backup.md (#6941)
MicrosoftDocs/windowsserverdocs: WindowsServerDocs/administration/windows-commands/auditpol-backup.md

---
title: auditpol backup
description: Reference article for the auditpol backup command that backs up system audit policy settings, per-user audit policy settings for all users, and all auditing options to a comma-separated value (CSV) text file.
ms.topic: reference
author: JasonGerend
ms.author: jgerend
ms.date: 01/03/2023
---
# auditpol backup

Applies to: Windows Server 2022, Windows Server 2019, Windows Server 2016, Windows Server 2012 R2, Windows Server 2012

The auditpol backup command backs up system audit policy settings, per-user audit policy settings for all users, and all auditing options to a comma-separated value (CSV) text file.

> To perform *backup* operations on the *per-user* and *system* policies, you need **Write** or **Full Control** permissions for that object set in the security descriptor. You can also perform *backup* operations if you have the **Manage auditing and security log** (SeSecurityPrivilege) user right. However, this right allows additional access that isn't necessary to perform the overall *backup* operations.

Reviewer: ```suggestion
To perform *backup* operations on the *per-user* and *system* policies, you need **Write** or **Full Control** permissions for that object set in the security descriptor. You can also perform *backup* operations if you've the **Manage auditing and security log** (SeSecurityPrivilege) user right. However, this right allows more access than necessary to perform the overall *backup* operations.
```
